### PR TITLE
Checklist in progress blocks

### PR DESCRIPTION
**Pull request by DaMishalkina**

### CODE REVIEW

File: src/partials/launch-subscribe.hbs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+               class="subscribe-container__header-icon">
+        {{/unless}}
+      </div>
+      <h2 class="subscribe-container__heading">
+        {{this.title}}
+      </h2>
+    </header>
+    <main>
+      {{#if this.checklists}}
+        <div class="subscribe-container__checklists-container checklists-container">
```
Comment (Zmoki) [inline src/partials/launch-subscribe.hbs:27]: this is a mix (https://ru.bem.info/methodology/quick-start/#%D0%BC%D0%B8%D0%BA%D1%81)
for mix, the block at first, than element
so, `checklists-container subscribe-container__checklists-container`

File: src/partials/launch-subscribe.hbs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          {{/each}}
+        </div>
+      {{/if}}
+
+      {{#each content}}
+        <p class="subscribe-container__content" id="paragraph-{{@index}}">
+          {{this.paragraph}}
+        </p>
+      {{/each}}
+      <div class="subscribe-container__form-wrapper email-form__wrapper">
```
Comment (Zmoki) [inline src/partials/launch-subscribe.hbs:58]: element must be inside block only

File: src/partials/launch-subscribe.hbs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        <form class="subscribe-container__form email-form">
```
Comment (Zmoki) [inline src/partials/launch-subscribe.hbs:59]: keep mix order

File: src/partials/launch-subscribe.hbs [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+          <label for="email" class="email-form__label">Your email</label>
+          <input type="email" id="email" class="email-form__input">
+        </form>
+        <a class="email-form__subscribe-button" href="">Subscribe</a>
+      </div>
+    </main>
+    <footer>
+      {{#each agreements}}
+        <p class="subscribe-container__agreement" id="">
```
Comment (Zmoki) [inline src/partials/launch-subscribe.hbs:68]: empty id? 🤔

**Comment by alexanderisora** [comment]
Great job! Looking forward to try it today 👍

**Pull request merged**